### ISSUE TITLE
Library Sync No Longer Working

### ISSUE THREAD

**Issue by IamMaris** [openaudible/openaudible#630]
My OpenAudible has stopped working for the Library Sync feature. I get an error message box with information that ends with "reason forbidden." I am attaching a screenshot.

The OpenAudible browser opens correctly and all information from my Audible library is shown correctly. I tried doing a refresh and it appeared to "refresh" correctly and the book count is correct, allowing for those that haven't downloaded as yet because I cannot sync the library with the quick or full options.

![OA error](https://user-images.githubusercontent.com/45048882/118143847-c76cbf00-b3d9-11eb-8dae-686859dffbd7.PNG)

**Comment by openaudible** [comment]
Can you Controls: log out of audible and log back in?

**Comment by IamMaris** [comment]
I followed your instruction to log out and for some reason, as it alerted me if what I was doing, I noticed it thought I was using an IPhone. I don't own an IPhone.

When I logged back in it asked if I wanted to do a full library refresh so I said yes and it's doing that right now and should be finishing up shortly.

It just finished the refresh and is asking me if I want to download the missing books and convert them so I said yes so it looks like it's all good again.

Thank you so much for your help.

Maris
